packages/functional-tests/lib/sms.ts

```diff
@@ -187,7 +192,11 @@ export class SmsClient {
         continue;
       }
 
-      const lastMessage = messages[0];
+      const emailLocalPart = email.split('@')[0];
+      const messagesWithEmailMetadata = messages.filter((message) =>
```

Comment:
Just double checking this. Do we want the last message, or the latest message? And which order is the list in?